lib/fluent/plugin/storage_local.rb

```diff
@@ -56,7 +56,7 @@ def configure(conf)
         elsif @path
           # ok
         else # @_plugin_id_configured is true
-          raise NotImplementedError, "implement this feature later with system_config"
+          raise NotImplementedError, "implement this feature later with system_config. Add <storage> section for file store or remove @id for memory store temporary"
```

Comment:
IMO, falling back to on-memory storage is better than adding configuration parameters for temporal issue.